doc/index.ru.rst

```diff
@@ -56,9 +56,9 @@
 
 
 .. _`Tarantool`:
-.. _`Tarantool homepage`: http://tarantool.org
-.. _`Tarantool/Box User Guide`: http://tarantool.org/tarantool_user_guide.html
-.. _`Mail.Ru`: http://mail.ru
+.. _`Tarantool homepage`: https://tarantool.io
+.. _`Документация Tarantool`: https://www.tarantool.io/en/doc/latest/
```

Comment:
We can refer to a russian version of docs here:
```suggestion
.. _`Документация Tarantool`: https://www.tarantool.io/ru/doc/latest/
```

Reply:
I was thinking about this, but we currently don't support Russian documentation. It looks like a patchwork of Russian and English text and may be hard to read for this reason. Are you sure you'd prefer the Russian link?

Reply:
Also, I want to squash the commits but it isn't enabled for this repository:
![image](https://user-images.githubusercontent.com/40490252/177356277-49186395-20bf-46dc-8711-04aa97b37704.png)

Reply:
> I was thinking about this, but we currently don't support Russian documentation. It looks like a patchwork of Russian and English text and may be hard to read for this reason. Are you sure you'd prefer the Russian link?

Well there is no problem. Let it stay 'en'.